src/model/types/index.ts

```diff
@@ -121,7 +193,7 @@ export type MeltResponse = {
 	/**
 	 * preimage of the paid invoice. can be null, depending on which LN-backend the mint uses
 	 */
-	preimage: string | null;
+	proof: string | null;
```

Comment:
It's the most generic attribute we came up with. For LN, this is the preimage, for any other payment method, it will be some kind of different proof (on-chain transaction for example).

Reply:
all I'm saying is, we're calling the ecash itself also a proof, which might lead to confusion down the road. But agreed, the other descriptions are not as fitting

Reply:
Oh yeah I see your point now and it's completely valid. However I would suggest that, now that we've fixed this in the protocol, we should implement it here as is and move this discussion to the NUTs repo!